common/beerocks/bwl/dwpal/sta_wlan_hal_dwpal.cpp

```diff
@@ -82,7 +82,7 @@ struct ConnectionStatus {
             case WiFiSec::WPA_WPA2_PSK:  return "WPA-WPA2-Personal";
 
             default: return "INVALID";
-        }    
```

Comment:
We would normally split this into 2 commits if you decided to fix whitespace issues - one for the actual change, and one for the whitespace fixes. However this is not crucial, just a comment - act on it or don't act on it, your choice (nitpicking)

(We mark comments which are not blockers for merge like this, @arnout uses a different OK icon)

Reply:
Got it, thanks for the comment.
Let me just push a commit on top, to have the CI job run go green before merging.